README.md

```diff
@@ -69,6 +69,7 @@ You can now use the plugin in any Vue file of your project as a component.
 | `showLabels`   | `Boolean`        | Optional  | `true`              | Show stream label in multiview mode.                                                                 |
 | `environment`  | `Object`         | Optional  | `.env file content` | Plugin environment. See [Environment options](#environment-options) on how to configure.             |
 | `mainLabel`    | `String`         | Optional  |                     | Allows to change the label of the main video.                                                        |
+| `startingQuality` | `String/Number` | Optional | Allows to start the stream at a specific resolution when available. |
```

Comment:
Add information of the possible values in the description. Also, type is String since query params works with strings.